libcxx/include/fstream

```diff
@@ -191,7 +191,8 @@ typedef basic_fstream<wchar_t> wfstream;
 #else
 #  include <__config>
 
-#  if _LIBCPP_HAS_FILESYSTEM && _LIBCPP_HAS_LOCALIZATION
```

Comment:
I'm seriously concerned that those ifs are being commented out. E.g., we do fiddle with `_LIBCPP_HAS_LOCALIZATION` on some of the resistant build environments. Instead of commenting this out, could a new variable been introduced and this conditional expression extended to make use of it?